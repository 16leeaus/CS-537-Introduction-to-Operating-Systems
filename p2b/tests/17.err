/bin/ls: cannot access 'new_1.out': No such file or directory
