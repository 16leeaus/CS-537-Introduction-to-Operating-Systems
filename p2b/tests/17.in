/bin/echo "this should print" > new_1.out
/bin/rm new_1.out
/bin/ls new_1.out
exit
/bin/echo "this should not print"
