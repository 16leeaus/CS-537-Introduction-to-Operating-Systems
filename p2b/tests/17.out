/bin/echo "this should print" > new_1.out
/bin/rm new_1.out
/bin/ls new_1.out
exit
